packages/components/src/sandbox/index.native.js

```diff
@@ -213,13 +214,19 @@ const Sandbox = forwardRef( function Sandbox(
 		// we can use this in the future to inject custom styles or scripts.
 		// Scripts go into the body rather than the head, to support embedded content such as Instagram
 		// that expect the scripts to be part of the body.
+
+		// Avoid comma issues with props.viewportProps.
+		const addViewportProps = viewportProps
+			.trim()
+			.replace( /(^[^,])/, ', $1' );
+
 		const htmlDoc = (
 			<html lang={ lang }>
 				<head>
 					<title>{ title }</title>
 					<meta
 						name="viewport"
-						content="width=device-width, initial-scale=1"
+						content={ `width=device-width, initial-scale=1 ${ addViewportProps }` }
```

Comment:
Nitpick: We can remove the extra space as extra viewport props will include the comma at the beginning of the string.
```suggestion
						content={ `width=device-width, initial-scale=1${ addViewportProps }` }
```

Reply:
It's interesting that with the extra space, I'm still getting the warning on Android when debugging with Chrome:
```
The key "" is not recognized and ignored.
```

Removing it as I suggested above makes the warning to not be shown.

Reply:
ugh, I had removed the space but it looked weird to me. I'll add the suggestion